conf/location-autocomplete-canonical-list.json

```diff
@@ -263,10 +263,6 @@
     "Czechia",
     "country:CZ"
   ],
-  [
-    "Czechoslovakia",
-    "country:CS"
-  ],
   [
     "Denmark",
     "country:DK"
```

Comment:
In this file you can find McDonald's Islands or something like this. As I remember in ticket we will have a new list with countries.

Reply:
covered in another ticket. Needs analysis. I have removed 4 obvious errors. Mcdonalds islands is a legit place.